src/main/java/com/internet/shop/dao/OrderDao.java

```diff
@@ -0,0 +1,18 @@
+package com.internet.shop.dao;
+
+import com.internet.shop.model.Order;
+import com.internet.shop.model.ShoppingCart;
+import java.util.List;
+import java.util.Optional;
+
+public interface OrderDao {
+    Order completeOrder(ShoppingCart shoppingCart);
```

Comment:
this operation doesn't related to CRUD, let's move this logic to service layer, her ewe should have add/create method